### ISSUE TITLE
'make' cannot find path to ceres.h

### ISSUE THREAD

**Issue by sydleither** [colmap/colmap#622]
Hi, I'm trying to install colmap on this server and I've had to install every dependency locally and after a long time of getting everything to work with 'cmake', when I finally do 'make' it says:

> [ 52%] Building CXX object src/CMakeFiles/colmap.dir/base/camera.cc.o
In file included from /CeresSolver/colmap/colmap/src/base/camera.cc:36:0:
/CeresSolver/colmap/colmap/src/base/camera_models.h:42:25: fatal error: ceres/ceres.h: No such file or directory
 #include <ceres/ceres.h>
                         ^
compilation terminated.
make[2]: *** [src/CMakeFiles/colmap.dir/base/camera.cc.o] Error 1
make[1]: *** [src/CMakeFiles/colmap.dir/all] Error 2
make: *** [all] Error 2

Now, when I ran 'cmake' I had to manually point the program to all the dependencies, and for ceres I did:

> cmake ../ -DCeres_DIR=/CeresSolver/ceres-solver/lib64/cmake/Ceres/

But when I run 'cmake' it says:

> -- Found Ceres version: 2.0.0 installed in: /CeresSolver/ceres-solver with components: [EigenSparse, SparseLinearAlgebraLibrary, SchurSpecializations]

So the path looks right to me, and I did check and ceres.h was in the right spot (ceres-solver/include/ceres/ceres.h). What should I change so the 'make' can finish successfully? Thank you.

Also just to make sure everything is clear- I'm on the 'make' step of "Configure and Compile COLMAP" from these instructions: https://colmap.github.io/install.html

**Comment by ahojnnes** [comment]
Not sure what’s happening here, but you could try to explicitly add the Ceres include directory to the include path.

**Comment by sydleither** [comment]
I tried that but now it's sending me on a path of adding the whole directory path to a new file's include statement every time I 'make'. I'm just going to manually add it to each file it errors on, but hopefully there's a better solution for anybody else who has this issue too.

**Comment by lod-uga** [comment]
Hello,
We meet exactly the same problem. Is there another solution than the one mentioned above? 
We also have at the level of the 'cmake' indicated the path to the dependencies,
The prerequisite seems good, the path is correct.
However when we launch 'make' it does not find the files and drops to 52% also with the same information as sydney.    the first missing file is ceres.h too.
If we manually modify the paths one by one in the different files, it seems it could works ... it's temporaly unlock the progress of the 'make' 
Sadly there are other files that are missing later, so we would like to ask if there another solution,
or a trick to resolve this problem.
thanks for your help.

**Comment by lod-uga** [comment]
In order to give precise details we'd like to add these informations : 
We specified the path to ceres-solver as follows:

> cmake .. -DCMAKE_INSTALL_PREFIX=/soft/x86_64/gcc_8.3.0/colmap_3.6
> -DCMAKE_PREFIX_PATH=/soft/x86_64/gcc_8.3.0

=> The result was 

> Found Ceres version: 2.0.0 installed in:
> /soft/x86_64/gcc_8.3.0/ceres-solver with components: [EigenSparse,
> SparseLinearAlgebraLibrary, LAPACK, SuiteSparse, CXSparse,
> SchurSpecializations]

But when we launch the 'make' it fails, and the reason given is the following:

> In file included from /tmp/colmap-3.6-dev.2/src/base/camera.cc:36:
> /tmp/colmap-3.6-dev.2/src/base/camera_models.h:42:10: fatal error:
> ceres/ceres.h: No such file or directory
>  #include <ceres/ceres.h>

Thanks for your help

**Comment by ahojnnes** [comment]
I am not sure what Ceres Version 2.0 is because the current stable version is 1.14?

**Comment by lod-uga** [comment]
Hello,
We would like to thank you for your help so fast and clear. The compilation works today.
We installed Ceres Version 1.14 and this solved the problem.
For those who encounter this problem, we tried to install colmap by following the documentation on this page 
`"https://colmap.github.io/install.html".`
On this page, in the part that describes the installation of ceres the command: 
`"git` clone `https://ceres-solver.googlesource.com/ceres-solver"`
is currently installing the problematic version.
Thanks a lot for your help.
